packages/ember-glimmer/lib/environment.js

```diff
@@ -62,6 +62,29 @@ function wrapClassAttribute(args) {
   return args;
 }
 
+function wrapClassBindingAttribute(args) {
+  let hasClassBinding = args.named.has('classBinding');
+
+  if (hasClassBinding) {
+    let { value , type } = args.named.at('classBinding');
+    if (type === 'value') {
+      let [ prop, truthy, falsy ] = value.split(':');
+      if (prop === '') {
+        let syntax = HelperSyntax.fromSpec(['helper', ['-class'], [truthy], null]);
+        args.named.add('class', syntax);
+      } else if (falsy) {
+        let parts = prop.split('.');
```

Comment:
@chadhietala in this case the object returned from `args.named.at('classBinding')` only has `type` and `value` properties. There's no `ref` property.